python_modules/dagster/dagster/components/component/component.py

```diff
@@ -100,7 +101,7 @@ def get_model_cls(cls) -> Optional[type[BaseModel]]:
 
     @classmethod
     def get_additional_scope(cls) -> Mapping[str, Any]:
-        return {}
+        return get_static_template_vars(cls)
```

Comment:
feels pretty easy to silently/accidentally override this in a subclass (e.g. user might not anticipate static template vars are "additional" scope)